db/migrate/20220603111112_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Kindly reference author instead of user

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Thanks a lot @Cathella for drawing my attention to this. The issue has been resolved